### DISCUSSION TITLE
`os.name` in Lua filter

### DISCUSSION THREAD

**Discussion by pr-apes** [speedata/publisher#483] [Q&A] [answered]
@pgundlach,

is there any way that I can use `os.name` in a filter?

`package.config:sub(1,1)` complains about `package:config` being nil (and this wouldn't allow distinctions inside Unix).

Many thanks for your help.

**Comment by pgundlach** [comment, marked as answer]
`os.name` is not available in the filter. Actually I don't use LuaTeX to interpret the filter file, I use https://github.com/yuin/gopher-lua. 

`print(package.config:sub(1,1))` works fine here.

**Comment by pr-apes** [comment]
Opening an issue.
